[jnyz-app/server]2024/03/13 - 21:02:00.135	[31merror[0m	/home/amos/repo/jnyz-app/server/core/server.go:54	accept tcp [::]:9999: use of closed network connection
[jnyz-app/server]2024/03/13 - 21:13:30.571	[31merror[0m	/home/amos/repo/jnyz-app/server/core/server.go:54	accept tcp [::]:9999: use of closed network connection
[jnyz-app/server]2024/03/13 - 21:16:11.600	[31merror[0m	/home/amos/repo/jnyz-app/server/core/server.go:54	accept tcp [::]:9999: use of closed network connection
[jnyz-app/server]2024/03/13 - 21:16:23.585	[31merror[0m	/home/amos/repo/jnyz-app/server/api/v1/system/sys_user.go:58	登陆失败! 用户不存在或者密码错误!	{"error": "record not found"}
[jnyz-app/server]2024/03/13 - 21:16:30.528	[31merror[0m	/home/amos/repo/jnyz-app/server/api/v1/system/sys_user.go:58	登陆失败! 用户不存在或者密码错误!	{"error": "密码错误"}
[jnyz-app/server]2024/03/13 - 21:17:34.874	[31merror[0m	/home/amos/repo/jnyz-app/server/core/server.go:54	accept tcp [::]:9999: use of closed network connection
[jnyz-app/server]2024/03/13 - 21:20:34.617	[31merror[0m	/home/amos/repo/jnyz-app/server/core/server.go:54	accept tcp [::]:9999: use of closed network connection
[jnyz-app/server]2024/03/13 - 21:22:10.482	[31merror[0m	/home/amos/repo/jnyz-app/server/core/server.go:54	accept tcp [::]:9999: use of closed network connection
[jnyz-app/server]2024/03/13 - 21:26:26.816	[31merror[0m	/home/amos/repo/jnyz-app/server/core/server.go:54	accept tcp [::]:9999: use of closed network connection
